aspnetcore/security/gdpr.md

```diff
@@ -17,6 +17,7 @@ ASP.NET Core provides APIs and templates to help meet some of the [EU General Da
 
 * The project templates include extension points and stubbed markup that you can replace with your privacy and cookie use policy.
 * The `Pages/Privacy.cshtml` page or `Views/Home/Privacy.cshtml` view provides a page to detail your site's privacy policy.
+* For GDPR guidance that applies to Blazor apps, see <xref:blazor/security/gdpr>.
```

Comment:
I would move this closer to the top if possible. Eventually the majority of the customers ramping up on ASP.NET Core will use Blazor, and we don't want to treat them as a secondary BTW thought..
Your call.

Reply:
@tdykstra will probably make such an update later because this is versioned with the list starting immediately at the top of the INCLUDE file.

Anyway, I just realized that it doesn't even make sense for the remark to be in the list. I just dropped it down to its own one-line paragraph under the list.